docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -130,14 +130,20 @@ The connector supports pushdown for a number of operations:
 * :func:`min`
 * :func:`sum`
 
-Limitations
------------
+.. _clickhouse-sql-support:
 
-The following SQL statements aren't  supported:
+SQL support
+-----------
 
-* :doc:`/sql/delete`
-* :doc:`/sql/grant`
-* :doc:`/sql/revoke`
-* :doc:`/sql/show-grants`
-* :doc:`/sql/show-roles`
-* :doc:`/sql/show-role-grants`
+The connector provides read and write access to data and metadata in
+a ClickHouse catalog. In addition to the :ref:`globally available
+<sql-globally-available>` and :ref:`read operation <sql-read-operations>`
+statements, the connector supports the following features:
```

Comment:
Should this be?
```suggestion
statements, the connector supports the following statements:
```

Reply:
See https://github.com/trinodb/trino/pull/8668#discussion_r678538095, leaving for now